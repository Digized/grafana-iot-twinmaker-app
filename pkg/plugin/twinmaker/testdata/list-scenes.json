{
    "NextToken": null,
    "SceneSummaries": [
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/AlarmWorkspace/scene/CookieFactory",
            "ContentLocation": "s3://twinmaker-workspace-alarmworkspace-166800769179-iad/CookieFactory.json",
            "CreationDateTime": "2022-04-27T17:30:08.759Z",
            "Description": null,
            "SceneId": "CookieFactory",
            "UpdateDateTime": "2022-04-27T17:30:08.759Z"
        }
    ]
}