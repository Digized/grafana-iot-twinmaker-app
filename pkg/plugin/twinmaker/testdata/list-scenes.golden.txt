ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: 
Dimensions: 4 Fields by 1 Rows
+------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+
| Name: arn                                                                                | Name: created                     | Name: description | Name: sceneId   |
| Labels:                                                                                  | Labels:                           | Labels:           | Labels:         |
| Type: []*string                                                                          | Type: []time.Time                 | Type: []*string   | Type: []*string |
+------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/AlarmWorkspace/scene/CookieFactory | 2022-04-27 17:30:08.759 +0000 UTC | null              | CookieFactory   |
+------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////aAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAHP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA8/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFz+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFQBAADYAAAAYAAAAAQAAADO/v//FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAALz+//8IAAAAEAAAAAcAAABzY2VuZUlkAAQAAABuYW1lAAAAAAAAAAC4/v//BwAAAHNjZW5lSWQAJv///xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAAU////CAAAABQAAAALAAAAZGVzY3JpcHRpb24ABAAAAG5hbWUAAAAAAAAAABT///8LAAAAZGVzY3JpcHRpb24AAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAADwAAABEAAAAAAAACkQAAAABAAAABAAAAIj///8IAAAAEAAAAAcAAABjcmVhdGVkAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABwAAAGNyZWF0ZWQAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEAAAABEAAAAAAAFAUAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAMAAAAAwAAAGFybgAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAADAAAAYXJuAAAAAAD/////SAEAABQAAAAAAAAADAAWABQAEwAMAAQADAAAAJAAAAAAAAAAFAAAAAAAAAMEAAoAGAAMAAgABAAKAAAAFAAAAMgAAAABAAAAAAAAAAAAAAALAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAIAAAAAAAAAFgAAAAAAAAAYAAAAAAAAAAAAAAAAAAAAGAAAAAAAAAACAAAAAAAAABoAAAAAAAAAAEAAAAAAAAAcAAAAAAAAAAIAAAAAAAAAHgAAAAAAAAAAAAAAAAAAAB4AAAAAAAAAAAAAAAAAAAAeAAAAAAAAAAIAAAAAAAAAIAAAAAAAAAADQAAAAAAAAAAAAAABAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAWAAAAGFybjphd3M6aW90dHdpbm1ha2VyOnVzLWVhc3QtMToxNjY4MDA3NjkxNzk6d29ya3NwYWNlL0FsYXJtV29ya3NwYWNlL3NjZW5lL0Nvb2tpZUZhY3RvcnnAq53cENHpFgAAAAAAAAAAAAAAAAAAAAAAAAAADQAAAENvb2tpZUZhY3RvcnkAAAAQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAABAABAAAAeAIAAAAAAABQAQAAAAAAAJAAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAHP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA8/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFz+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFQBAADYAAAAYAAAAAQAAADO/v//FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAALz+//8IAAAAEAAAAAcAAABzY2VuZUlkAAQAAABuYW1lAAAAAAAAAAC4/v//BwAAAHNjZW5lSWQAJv///xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAAU////CAAAABQAAAALAAAAZGVzY3JpcHRpb24ABAAAAG5hbWUAAAAAAAAAABT///8LAAAAZGVzY3JpcHRpb24AAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAADwAAABEAAAAAAAACkQAAAABAAAABAAAAIj///8IAAAAEAAAAAcAAABjcmVhdGVkAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABwAAAGNyZWF0ZWQAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEAAAABEAAAAAAAFAUAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAMAAAAAwAAAGFybgAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAADAAAAYXJuAJACAABBUlJPVzE=
